pyvista/ext/OfflineLocalView.html

```diff
@@ -0,0 +1,13 @@
ed()}if(null!==t.renderable.getJsImageData()){var a=t.renderable.getJsImageData();t.renderable.getInterpolate()&&(t.generateMipmap=!0,e.setMinificationFilter(Gd.LINEAR_MIPMAP_LINEAR)),e.create2DFromRaw(a.width,a.height,4,Ud.UNSIGNED_CHAR,a.data,!0),e.activate(),e.sendParameters(),t.textureBuildTime.modified()}var o=t.renderable.getInputData(0);if(o&&o.getPointData().getScalars()){for(var i=o.getExtent(),s=o.getPointData().getScalars(),u=[],c=0;c<t.renderable.getNumberOfInputPorts();++c){var l=t.renderable.getInputData(c),f=l?l.getPointData().getScalars().getData():null;f&&u.push(f)}t.renderable.getInterpolate()&&4===s.getNumberOfComponents()&&(t.generateMipmap=!0,e.setMinificationFilter(Gd.LINEAR_MIPMAP_LINEAR)),u.length%6==0?e.createCubeFromRaw(i[1]-i[0]+1,i[3]-i[2]+1,s.getNumberOfComponents(),s.getDataType(),u):e.create2DFromRaw(i[1]-i[0]+1,i[3]-i[2]+1,s.getNumberOfComponents(),s.getDataType(),s.getData()),e.activate(),e.sendParameters(),t.textureBuildTime.modified()}}t.handle&&e.activate()},e.destroyTexture=function(){e.deactivate(),t.context&&t.handle&&t.context.deleteTexture(t.handle),t.handle=0,t.numberOfDimensions=0,t.target=0,t.components=0,t.width=0,t.height=0,t.depth=0,e.resetFormatAndType()},e.createTexture=function(){t.handle||(t.handle=t.context.createTexture(),t.target&&(t.context.bindTexture(t.target,t.handle),t.context.texParameteri(t.target,t.context.TEXTURE_MIN_FILTER,e.getOpenGLFilterMode(t.minificationFilter)),t.context.texParameteri(t.target,t.context.TEXTURE_MAG_FILTER,e.getOpenGLFilterMode(t.magnificationFilter)),t.context.texParameteri(t.target,t.context.TEXTURE_WRAP_S,e.getOpenGLWrapMode(t.wrapS)),t.context.texParameteri(t.target,t.context.TEXTURE_WRAP_T,e.getOpenGLWrapMode(t.wrapT)),t._openGLRenderWindow.getWebgl2()&&t.context.texParameteri(t.target,t.context.TEXTURE_WRAP_R,e.getOpenGLWrapMode(t.wrapR)),t.context.bindTexture(t.target,null)))},e.getTextureUnit=function(){return t._openGLRenderWindow?t._openGLRenderWindow.getTextureUnitForTexture(e):-1},e.activate=function(){t._openGLRenderWindow.activateTexture(e),e.bind()},e.deactivate=function(){t._openGLRenderWindow&&t._openGLRenderWindow.deactivateTexture(e)},e.releaseGraphicsResources=function(n){n&&t.handle&&(n.activateTexture(e),n.deactivateTexture(e),t.context.deleteTexture(t.handle),t.handle=0,t.numberOfDimensions=0,t.target=0,t.internalFormat=0,t.format=0,t.openGLDataType=0,t.components=0,t.width=0,t.height=0,t.depth=0),t.shaderProgram&&(t.shaderProgram.releaseGraphicsResources(n),t.shaderProgram=null)},e.bind=function(){t.context.bindTexture(t.target,t.handle),t.autoParameters&&e.getMTime()>t.sendParametersTime.getMTime()&&e.sendParameters()},e.isBound=function(){var e=!1;if(t.context&&t.handle){var n=0;t.target===t.context.TEXTURE_2D?n=t.context.TEXTURE_BINDING_2D:Wd("impossible case"),e=t.context.getIntegerv(n)===t.handle}return e},e.sendParameters=function(){t.context.texParameteri(t.target,t.context.TEXTURE_WRAP_S,e.getOpenGLWrapMode(t.wrapS)),t.context.tex
```

Comment:
Let's hope there's no typo on line 10 :smile: I know this is a draft still, but do you plan to add some comment in this HTML to explain what this does, where it's from, what generates it and so on?

Reply:
This is autogenerated by VTK.js from: https://kitware.github.io/vtk-js/examples/OfflineLocalView.html (no typos/bugs since this is generated with webpack)

I may actually try to move this to trame-vtk so that the version is consistent and this build doesn't go stale when trame-vtk updates.

Reply:
Does this need to be versioned, then? Could we generate it on demand instead? Might not make sense, just wondering.

Reply:
It may need to be versioned... I think it makes the most sense to have `trame-vtk` generate it per release

Reply:
Neat, thanks! FYI "emb[b]ed" has a typo in the new trame helper.

Reply:
Thanks for pointing that out! Looks like that slipped by a couple of us!